src/components/team-gallery/team-gallery.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <section class="grid">
+    <ul class="team-gallery__list">
+      <li
+        v-for="member in team "
+        :key="member.id"
+        class="team-gallery__list-item"
+      >
+        <app-link
+          :to="$localeUrl({ name: 'team-slug', params: { slug: member.slug } })"
+          class="team-gallery-member__link"
+          tabindex="-1"
+        >
+          <dato-image
+            class="team-gallery-member__image"
+            :src="member.image.url"
+            :alt="member.name"
+            :width="member.image.width"
+            :height="member.image.height"
+            loading="lazy"
+            sizes="(min-width: 800px) 20vw, (min-width: 500px) 33vw, 50vw"
+          />
+        </app-link>
+        <div class="team-gallery-member__details">
+          <app-link
```

Comment:
Could you maybe put an `::after` element on here that positions absolutely over the entire list item to make the entire item clickable instead of wrapping the image in an `app-link` with `tabindex="-1"`? I think the fewer accessibility hacks we have to apply, the better.

Reply:
✅